### DISCUSSION TITLE
LVM Partitions

### DISCUSSION THREAD

**Discussion by fmonera** [mysticaltech/terraform-hcloud-kube-hetzner#655] [Q&A] [answered]
I would like to use TopoLVM and Rook-Ceph to manage the free disk space after operating system installation.

Is there any way to setup the system so that the disk is partitioned using LVM? Or at least, use the first 20Gb for the Operating System partition, and leave the rest unpartitioned.

What would be the correct approach for this?

**Comment by mysticaltech** [comment, marked as answer]
@fmonera We use MicroOS and it's super optimized, maybe you can find your answer here https://en.opensuse.org/Portal:MicroOS. 

For the moment we support Longhorn which you can also configure to use node storage, I suggest you try that instead as it's already proven to work well. Please see the kube.tf.example file, and also search this repo.

**Comment by fmonera** [comment]
Ok. I really needed snapshot volumes for velero backups. But I see that Longhorn implements that. 
Thanks, that solves my immediate problem.
